More Context info (#4)
hasheddan/crossplane-offline: docs/WORKSHOP.md

- `hasheddan/k8scr-distribution:latest`: this image is used to run a registry in
  the `kind` cluster that users can push `Configuration` image to.

> In addition we will pull (and later push) an image for the Crossplane package manger to use:

Reviewer: ```suggestion
In addition we will pull (and later push) an image for the Crossplane package manager to use:
```

helm pull crossplane-stable/crossplane --version 1.4.1
```

### 3. Download `k8scr` CLI.

> For the Crossplane package manager we need to install a local container

Reviewer: nitpick: local could indicate that we are running it locally outside of cluster
```suggestion
For the Crossplane package manager we need to install an in-cluster container
```

```
kubectl get pods -n crossplane-system
NAME                                       READY   STATUS    RESTARTS   AGE
crossplane-556965f687-69gd8                1/1     Running   0          7s
crossplane-rbac-manager-55b5cb98f7-gzqkr   1/1     Running   0          7s
```

### 4. Install Localstack.

```
helm install localstack ./localstack-0.3.4.tgz --set startServices="s3" --set image.pullPolicy=Never
```

You can check the localstack pod and it's logs:
```
kubectl get pods
NAME                         READY   STATUS    RESTARTS   AGE
k8scr                        1/1     Running   0          3m34s
localstack-5d679bf9f-l72lm   1/1     Running   0          21s

kubectl logs $(kubectl get pods -l app.kubernetes.io/name=localstack -o name) -f
Waiting for all LocalStack services to be ready
2021-10-25 16:53:43,561 CRIT Supervisor is running as root.  Privileges were not dropped because no user is specified in the config file.  If you intend to run as root, you can set user=root in the config file to avoid this message.
2021-10-25 16:53:43,570 INFO supervisord started with pid 29
```

### 4. Push `provider-aws:v0.20.0` to in-cluster registry.

> This will push the provider-aws inot into the `k8scr` registry:

Reviewer: ```suggestion
This will push the `provider-aws` package image into the `k8scr` registry:
```
